common/src/main/java/com/wynntils/models/wynnitem/parsing/WynnItemParser.java

```diff
@@ -55,6 +58,10 @@ public final class WynnItemParser {
 
     public static final Pattern SET_BONUS_PATTEN = Pattern.compile("^§aSet Bonus:$");
 
+    // Test suite: https://regexr.com/7i5h5
+    public static final Pattern SHINY_STAT_PATTERN = Pattern.compile("^§f⬡ §7([a-zA-Z ]*?): §f(\\d+)$");
+    public static final Pattern SHINY_STAT_NO_CAPTURE_PATTERN = Pattern.compile("^§f⬡ §7[a-zA-Z ]*?: §f\\d+$");
```

Comment:
What's the point of having the no capture pattern?

Reply:
there are cases where the capture return values are discarded, should I still use the capture pattern anyways? I am new to using regex.

Reply:
capture groups are only relevant if you actually get them. you can just ignore them existing, if you dont need them.

Reply:
okay, thx for letting me know.